src/applications/claims-status/components/claim-document-request-pages/Automated5103Notice.jsx

```diff
@@ -0,0 +1,65 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Link } from 'react-router-dom-v5-compat';
+import { buildDateFormatter } from '../../utils/helpers';
+
+function Automated5103Notice({ item }) {
```

Comment:
You can just put the export default up here, that's part of the reason we generally use `function` instead of `const` for functional components

Reply:
Updated this and the DefaultPage component! Please re-review @jerekshoe !